municipal-services/land-services/pom.xml

```diff
@@ -94,7 +98,7 @@
 		<dependency>
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
-			<version>42.2.2.jre7</version>
+			<version>42.2.18</version>
```

Comment:
Revert this change

Reply:
if the version is 42.2.2.jer7 it getting Jdbc template error.

Reply:
Done vinoth